scripts/docker/hub/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ubuntu:xenial
-MAINTAINER Parity Technologies <devops@parity.io>
+LABEL MAINTAINER="Parity Technologies <devops@parity.io>"
```

Comment:
if changing this would be nice to change it even to "Parity Technologies <devops-team@parity.io>"